tasks.yaml

```diff
@@ -4,34 +4,53 @@ variables:
     # x-release-please-start-version
     default: "0.3.0"
     # x-release-please-end
+  - name: IMAGE_NAME
+    description: "The repository + name for the published image"
+    default: "ghcr.io/defenseunicorns/uds/identity-config"
 
 tasks:
   - name: build-and-publish
+    description: "Build and publish the multi-arch image"
     actions:
-      - cmd: |
-          docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ghcr.io/defenseunicorns/uds/identity-config:${VERSION} src
+      - cmd: docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ${IMAGE_NAME}:${VERSION} src
+
+  - name: dev-build
+    description: "Build the image locally for dev"
+    actions:
+      - cmd: docker build src -t uds-core-config:keycloak
 
   - name: dev-update-image
+    description: "Build the image and import locally into k3d"
     actions:
+      - task: dev-build
       - cmd: |
-          docker build src -t uds-core-config:keycloak
           k3d image import -c uds uds-core-config:keycloak
           kubectl rollout restart statefulset -n keycloak keycloak
 
   - name: dev-theme
+    description: "Copy theme to Keycloak in dev cluster"
     actions:
-      - description: "Copy theme to Keycloak (must be run from src/keycloak dir)"
-        cmd: |
+      - cmd: |
           PV=$(kubectl get pvc keycloak-themes -n keycloak -o jsonpath='{.spec.volumeName}')
           THEME_PATH=$(kubectl get pv $PV -o jsonpath="{.spec.hostPath.path}")
           docker cp src/theme k3d-uds-server-0:/$THEME_PATH
 
+  - name: cacert
+    description: "Get the CA cert value for the Istio Gateway"
+    actions:
+      # This is written to a file rather than printed because it is a massive value to copy out of the terminal
+      - cmd: |
+          docker run --rm --entrypoint sh ${IMAGE_NAME}:${VERSION} -c "cat /home/nonroot/authorized_certs.pem | base64 -w 0" > cacert.b64
```

Comment:
i think we could also do this in the script itself as a flag if that's less ugly? I dont have strong prefs either way.

Reply:
Going to punt on that for now to get this across the line and cut a release of the image, but I think it would make things slightly nicer. Ideally no one besides us/developers cares about the command here, since they are just doing a `uds run` command.